gapic/src/main/com/google/gapid/perfetto/models/GpuInfo.java

```diff
@@ -110,6 +125,26 @@ public String getDisplay() {
     }
   }
 
+  public static class VkEvent {
```

Comment:
This needs a different name. VkEvent is an object in the Vulkan API and so this is going to cause confusion.

Reply:
Let's go with VulkanApiEvent or VkApiEvent. I am fine with full name or abbreviation.

Reply:
Done in a new commit. VkEvent -> VkApiEvent.